Sources/Orbit/Components/Text.swift

```diff
@@ -254,6 +265,16 @@ extension TextAlignment {
     }
 }
 
+// MARK: - TextRepresentable
+extension Text: TextRepresentable {
+
+    public func swiftUITextContent(configuration: ContentSizeCategory) -> SwiftUI.Text? {
+        guard content.isEmpty == false else { return nil }
```

Comment:
Same as the other file.

Reply:
Copied from previous implementation, updated.